commands/chassis/ChassisShift.java

```diff
@@ -13,7 +13,7 @@
 	protected final SolenoidShifters.ShiftState state;
 	
 	/**
-	 * Shifts the solenoids to the state state
+	 * Shifts the solenoids to the opposite state
```

Comment:
I would recommend that this comment say something like `Shifts the solenoids to the given state` and that the one on the toggle constructor says something along the lines of `Toggles the solenoids`